### PR TITLE
feat: previously stopped task with the stopped task id

### LINKED ISSUE #3272: Proposal: `logs -p` or `logs --open`
Currently, if I want to check for the logs of exited tasks, I need to go to AWS console, navigate to ECS, find the cluster, then the service, than switch to stopped tasks, then search for the logs there.

It would be helpful to have a `copilot svg logs -p` where `-p` stands for "previous task" (like k8s), that dumps the logs of the most recent stopped task.

Besides that, an `copilot svc logs --open` that open the cloudwatch logs console with filter applied would be helpful too, just some extra steps (like login in).

### PR DESCRIPTION
**Pull request by paragbhingre**
Fixes #3272 and #3961, reverting #3960

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/svc_logs.go [hunk 1]
```diff
@@ -209,6 +209,7 @@ func (o *svcLogsOpts) Execute() error {
 			return err
 		}
 		o.taskIDs = []string{taskID}
+		fmt.Printf("previously stopped task: %s\n", taskID)
```
Comment (huanjani) [inline internal/pkg/cli/svc_logs.go]: delete?

File: internal/pkg/cli/svc_logs_test.go [hunk 2]
```diff
@@ -106,6 +108,14 @@ func TestSvcLogs_Validate(t *testing.T) {
 
 			wantedError: fmt.Errorf("--limit 10001 is out-of-bounds, value must be between 1 and 10000"),
 		},
+		"returns error if both previous and tasks flags are defined": {
```
Comment (huanjani) [inline internal/pkg/cli/svc_logs_test.go:112]: Do we also have any unit tests for success with taskID(s)?

Comment (paragbhingre) [reply internal/pkg/cli/svc_logs_test.go:112]: Yes, I had implemented that unit test as a part of original ask of --previous flag. You can check the unit test [here](https://github.com/aws/copilot-cli/blob/e1ae307bda11ab8d8e940f9fefd398ced094bb16/internal/pkg/cli/svc_logs_test.go#L374)

File: internal/pkg/cli/svc_logs.go (hunk 1 above)
Comment (paragbhingre) [reply internal/pkg/cli/svc_logs.go]: Ohh so we discussed in the meeting that we want to show taskId too as a part of `--previous` functionality. That is why I am displaying taskId before we show logs for the that stopped task.

Comment (huanjani) [reply internal/pkg/cli/svc_logs.go]: Ah, gotcha. I was thrown off by the `fmt.Printf`...should we use a `log.` instead?

**Review by huanjani — APPROVED** [review]
Looks good. One nit, then feel free to remove label!

**Review by Lou1415926 — APPROVED** [review]
LGTM! and +1 to @huanjani about using `log`. Does this fix #3272 though? Did you mean to link a different issue?

**Comment by paragbhingre** [comment]
> LGTM! and +1 to @huanjani about using `log`. Does this fix #3272 though? Did you mean to link a different issue?

We had reverted the changes for the fix #3272 in #3960 right? So this change adds those changes back along with printing task id. Does that make sense?

**Pull request merged**